tests/Class1.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+
+namespace MGroup.Stochastic.Structural.Example
+{
+    public class SolveProblem
+    {
+        public void Solve()
+        {
+            const int iterations = 1000;
+            const double youngModulus = 2.1e8;
+
+            var domainMapper = new CantileverStochasticDomainMapper(new [] { 0d, 0d, 0d });
+            var realizer = new GiannisStructuralStochasticRealizer(youngModulus, domainMapper);

```

Comment:
Rename the **GiannisStructuralStochasticRealizer** and **Class1** with a more suitable names.

Reply:
done